src/lib/components/event/event-group-details.svelte

```diff
@@ -1,18 +1,22 @@
 <script lang="ts">
   export let eventGroup: CompactEventGroup | null;
-  export let eventId: string;
-  export let onClick: (id: string) => void;
+  export let selectedId: string;
 </script>
 
-<section class="overflow-y-scroll max-h-full">
+<section class="overflow-y-scroll max-h-full p-4">
   {#if eventGroup}
     <nav class="flex flex-col mb-4">
       <ul class="gap-4 w-full items-start">
         {#each [...eventGroup.events] as [id, eventInGroup] (id)}
-          <li on:click|preventDefault|stopPropagation={() => onClick(id)}>
-            <span class="event-type" class:active={id === eventId}
+          <li
+            on:click|preventDefault|stopPropagation={() => {
+              selectedId = id;
+            }}
+          >
+            <span class="event-type" class:active={id === selectedId}
               >{eventInGroup.eventType}</span
             >
+            ({id})
```

Comment:
Don't think this needs to be changed, but I thought it was a count and not an id. Just food for thought

Reply:
What if we put a “#” before it?